examples/07_advanced/plot_copy_headers_math_img.py

```diff
@@ -0,0 +1,146 @@
+"""
+Copying headers from input images with ``math_img``
```

Comment:
Moved it from `06_manipulating_images` to `07_advanced` as @Remi-Gau said in the dev meeting